options/options.css

```diff
@@ -0,0 +1,12 @@
+form#titler_options {
```

Comment:
Use kebab case instead of snake case for html tag ids/classes to stay consistent in this project

Reply:
ok